src/hooks/usePrevNextArticle.ts

```diff
@@ -16,14 +16,19 @@ const usePrevNextArticle = () => {
     }, 50);
   });
 
-  const getNextArticle = () => {
-    let a = document.querySelector(".next_page") as HTMLAnchorElement;
-    a?.click();
+  const getNextArticle = async () => {
```

Comment:
`setTimeout` is a hack and should not be used! You should use `requestAnimationFrame` or `onMount` logic. There is better way to do it.